Source/Core/InputCommon/ControllerEmu/ControlGroup/Cursor.cpp

```diff
@@ -62,6 +62,8 @@ Cursor::Cursor(std::string name_, std::string ui_name_)
 
   AddSetting(&m_relative_setting, {_trans("Relative Input")}, false);
   AddSetting(&m_autohide_setting, {_trans("Auto-Hide")}, false);
+  numeric_settings.back().get()->GetInputReference().default_range = AUTO_HIDE_MS / 1000.0;
+  numeric_settings.back().get()->GetInputReference().range = AUTO_HIDE_MS / 1000.0;
```

Comment:
This is setting the default value, which I believe was called range. But I'm not sure now I'm from my phone, will look tomorrow.

Reply:
Right, no, this is useless, I think I was just testing whether the range would be visualized correctly in the UI and then forgot the code wasn't doing anything. Though I had hoped to just set the numeric setting range, and then read that back instead of reading AUTO_HIDE_MS in the cursors update, so that if we ever introduced range on fixed input numeric settings, it would already be changeable. Anyway doesn't matter.